subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -657,12 +657,57 @@ config BT_CTLR_GPIO_LNA_POL_INV
 config BT_CTLR_GPIO_LNA_OFFSET
 	int "Time from LNA ON to Rx ready"
 	default 5
-	range 0 10
+	range 0 15
 	help
 	  Time before Rx ready to turn on LNA.
 
 endif # BT_CTLR_GPIO_LNA
 
+menuconfig BT_CTLR_FEM_NRF21540
+	bool "nRF21540 GPIO interface"
+	depends on !SOC_SERIES_NRF51X
+	select BT_CTLR_GPIO_PA
+	select BT_CTLR_GPIO_LNA
+	help
+	  Enable PDN and CSN GPIO interface for the nRF21540. This allows
+	  hardware designs using the nRF21540 to let the controller toggle
+	  their state based on radio activity.
+
+if BT_CTLR_FEM_NRF21540
+
+config BT_CTLR_GPIO_PDN_PIN
```

Comment:
Forgive my ignorance, but are these Kconfig options being copy/pasted from somewhere else for a different SoC?

If not, they should definitely be in DT instead, along with the polarity (since we have the GPIO_ACTIVE_LOW flag in DTS), CSN pin, etc.

Reply:
I am not against use of DT, it is just that original controller contribution was before use of DT, and not all of controller has been ported to DT use.

> they should definitely be in DT

Agree. As a subsequent PR, please do work on the port.